pages/guides/developing/upload-a-video-asset.en-US.mdx

```diff
@@ -121,6 +121,38 @@ export const CreateAndViewAsset = () => {
 };
 ```
 
+### Upload video to IPFS
```

Comment:
```suggestion
### Step 3: Upload video to IPFS
```

Let's keep the formatting of the existing guide.

Reply:
(And change step3 to step4 below)

Reply:
I thought, it would be an optional section for developer who just want to directly upload videos IPFS, but make sense. I will update it. Thanks!!